styles/designs/carnival/parts/_columns-shapes.scss

```diff
@@ -24,10 +26,14 @@
                 $Columns--FourColumnsFiveRatioTwo,
                 $Columns--FourColumnsSixRatioOne,
                 $Columns--FourColumnsSixRatioThree,
+                $Columns--FourColumnsSevenRatioOne,
+                $Columns--FourColumnsSevenRatioTwo,
                 $Columns--FourColumnsEightRatioOne,
                 $Columns--FourColumnsEightRatioTwo,
                 $Columns--FourColumnsEightRatioThree,
                 $Columns--FiveColumnsFiveRatioOne,
+                $Columns--FiveColumnsNineRatioOne,
+                $Columns--FiveColumnsNineRatioTwo,
```

Comment:
I see 9 new components added to the `columns-components` file but only 6 added to the shape. what's going on with the un-added ones?

Reply:
thanks for catching this, remaining components have been added.